### ISSUE TITLE
Field rendering in layer list

### ISSUE THREAD

**Issue by kakada** [ilabsea/resourcemap-program-assessment#71]
1. Go to layer tab
2. Layer list only 10 field for each even it has more than 10 fields
3. Click edit -> click cancel
4. Layer list all fields -> Browser will getting hang if layer has about more than 1k fields(WFP case)
![layer_list_field](https://cloud.githubusercontent.com/assets/745429/20093083/d5178f46-a5cd-11e6-96b1-b90aafd0bab1.png)

**Comment by sokmesakhiev** [comment]
Layer index will list only first 10 fields and tell the number of total field above Edit button. To see all field please click on number of total field. We make this because we improve rendering speed when each layer have many field.

**Comment by kakada** [comment]
Yeah, it should list only first 10 fields but here it list all of fields that will make browser getting hang if the number of field is getting bigger and bigger

**Comment by sokmesakhiev** [comment]
From what we test bottle neck of the layer, we limit the number of field up to 200 fields per layer and more than that we will not guarantee that the application can run.
